src/flagStore.ts

```diff
@@ -64,13 +66,43 @@ export class FlagStore {
 			const ldConfig = this.ldConfig();
 			const ldClient = await LaunchDarkly.init(sdkKey, ldConfig).waitForInitialization();
 			this.resolveLDClient(ldClient);
+			this.on('update', async flag => {
+				try {
+					const updatedFlag = await this.getFeatureFlag(flag.key);
+					this.storeUpdates.fire(updatedFlag.key);
+				} catch (err) {
+					console.error('Failed to update LaunchDarkly flag store.', err);
+				}
+			});
+			if (this.config.refreshCron) {
+				if (this.config.validateCron(this.config.refreshCron)) {
+					this.cron(this.config.refreshCron);
+				} else {
+					window.showErrorMessage(
+						`Invalid cron expression: '${this.config.refreshCron}' needs to be 5 field cron format.`,
+					);
+				}
+			}
 		} catch (err) {
 			this.rejectLDClient();
 			console.error(err);
 		}
 	}
 
-	async on(event: string, cb: FlagUpdateCallback) {
+	private async cron(exp: string) {
+		var CronJob = cron.CronJob;
+		var job = new CronJob(
+			exp,
+			async () => {
+				this.updateFlags();
+			},
+			null,
+			true,
+		);
+		job.start();
```

Comment:
cron package seemed much more reliable and flexible for users. Though with just minutes it might be possible. I'll first look into just converting minutes to a cron expression and get everything working there. As a future much smaller refactoring I can look to remove this package.